workbench-core/authentication/src/index.ts

```diff
@@ -32,7 +32,7 @@ export { UserAlreadyExistsError, isUserAlreadyExistsError } from './errors/userA
 export { UserNotFoundError, isUserNotFoundError } from './errors/userNotFoundError';
 export { DecodedJWT } from './decodedJWT';
 export { Tokens } from './tokens';
-export { getTimeInSeconds } from './utils';
+export { getTimeInSeconds, TimeUnits } from './utils';
```

Comment:
this import needed?

Reply:
Its the enum used as a param in the `getTimeInSeconds` function, so it made sense to export it as well.

Reply:
oh sorry your right this is your index file